components/shared_code/src/shared_data_model/metrics.py

```diff
@@ -17,132 +17,131 @@
     "https://semver.org",
 ]
 
-METRICS = Metrics.parse_obj(
```

Comment:
ok I'm gonna assume that nothing weird happened here!

Reply:
Correct. Instead of parsing Python dicts, we simply instantiate the models.